File: Misc/Unknown_Title.java
/**
No statement found
*/

class Solution {
    public int threeSumClosest(int[] nums, int target) {
        Arrays.sort(nums);
        int closest_sum = Integer.MAX_VALUE / 2;  // A large value but not overflow
        
        for (int i = 0; i < nums.length - 2; ++i) {
            int left = i + 1, right = nums.length 

File: Misc/Unknown_Title.py
"""
No statement found
"""

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        lst = []
        for i in range (0,length):
            for j in range (i+1, length):
                if(nums[i]+nums[j] == target):
                    lst.append(i)
                    lst.append(j)
        return lst